charts/merlin/templates/_helpers.tpl

```diff
@@ -37,7 +37,7 @@ Generated names
 {{- end -}}
 
 {{- define "merlin.envs-cm-name" -}}
-    {{- printf "%s-environments" (include "merlin.resource-prefix" .) | trunc 63 | trimSuffix "-" -}}
+    {{- printf "%s-config" (include "merlin.resource-prefix-with-release-name" .) | trunc 63 | trimSuffix "-" -}}
```

Comment:
Maybe let's have it similar to authz bootstrap: `release-environments-config` because I anticipate that we will have a `release-config` for merlin in future as well as we have for mlp

Reply:
I think, it's better to leave it this way, since, environments are just one of the data stored in that config, if it needs to be extended, one can add more keys in the same configMap. wdyt?

```
data:
  environment.yaml: |-
```

Reply:
fair point